docs/contributors.rst

```diff
@@ -55,11 +55,17 @@ From *inside* the Horovod root directory, remove any previous build artifacts an
 .. code-block:: bash
 
     $ rm -rf build/ dist/
-    $ HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 python setup.py install
+    $ HOROVOD_WITH_PYTORCH=1 \
+      HOROVOD_WITH_TENSORFLOW=1 \
+      HOROVOD_BUILD_IN_REPO="$PWD" \
+      python setup.py install
 
 Set ``HOROVOD_WITHOUT_[FRAMEWORK]=1`` to disable building Horovod plugins for that framework.
 This is useful when you’re testing a feature of one framework in particular and wish to save time.
 
+The ``HOROVOD_BUILD_IN_REPO`` enables incremental rebuilds by storing the intermediate build artifacts
```

Comment:
What does "in repo" refer to here? The "source repository", which is the git clone directory?

Reply:
I am not forced to provide a path to the sources here, right? Can be any directory, so the name is unnecessarily restrictive.

Reply:
Sure.  I'm down for any name you see fit.  Maybe `HOROVOD_PERSISTENT_BUILD_DIR`?

Reply:
Wait, actually, unless you also specify the `--use-feature=in-tree-build` feature, you actually do need the source to the repository in order for this to work, because if you point cmake at the copied source code directory that pip creates, I think you won't get the right timestamps and you'll always be rebuilding everything anyway.

Reply:
maybe something like `HOROVOD_SOURCE_DIR`